R/ErrorHandling.R

```diff
@@ -134,6 +134,7 @@ bptry <- function(expr, ..., bplist_error, bperror)
 .error_bplist <- function(result) {
     idx <- which(!bpok(result) &
                   !vapply(result, inherits, logical(1), "not_available_error"))
+        # !vapply(result, inherits, logical(1), "unevaluated_error")
```

Comment:
I originally want to ignore the `unevaluated_error` in `.error_bplist` as they are due to the error that occurs in the previous step and are confusing, but I think it might be better to do it in another pull request so I comment it out. I'll remove this line in the next commit.